apps/web/src/services/dashboards/widgets/asset-widgets/total-failure-and-severity/TotalFailureAndSeverityWidget.vue

```diff
@@ -255,22 +224,50 @@ const state = reactive({
 const widgetFrameProps:ComputedRef = useWidgetFrameProps(props, state);
 
 /* API */
-const fetchData = async (): Promise<Data[]> => {
+const apiQueryHelper = new ApiQueryHelper();
+const fetchTrendData = async (): Promise<CloudServiceStatsModel[]> => {
     try {
-        const apiQueryHelper = new ApiQueryHelper();
         apiQueryHelper
             .setFilters(state.consoleFilters)
-            .addFilter({ k: 'ref_cloud_service_type.labels', v: 'Compliance', o: '=' });
-        const { results } = await SpaceConnector.clientV2.inventory.cloudService.analyze({
+            .addFilter({ k: 'ref_cloud_service_type.labels', v: 'Compliance', o: '=' })
+            .addFilter({ k: 'key', v: ['fail_finding_count'], o: '' });
+        const { results } = await SpaceConnector.clientV2.inventory.cloudServiceStats.analyze({
             query: {
-                group_by: ['data.severity'],
+                granularity: 'MONTHLY',
+                start: state.dateRange.start,
+                end: state.dateRange.end,
+                group_by: ['key', 'unit'],
                 fields: {
-                    fail_finding_count: {
-                        key: 'data.stats.checks.fail',
+                    value: {
+                        key: 'value',
                         operator: 'sum',
                     },
-                    total_finding_count: {
-                        key: 'data.stats.checks.total',
+                },
+                ...apiQueryHelper.data,
+            },
+        });
+        return results;
+    } catch (e) {
+        ErrorHandler.handleError(e);
+        return [];
+    }
+};
+const fetchRealtimeData = async (): Promise<CloudServiceStatsModel[]> => {
+    try {
+        apiQueryHelper
+            .setFilters(state.consoleFilters)
+            .addFilter({ k: 'ref_cloud_service_type.labels', v: 'Compliance', o: '=' })
+            .addFilter({ k: 'key', v: ['fail_finding_count', 'pass_findings_count'], o: '' });
+        const prevMonth = dayjs.utc(state.settings?.date_range?.start).subtract(1, 'month').format(DATE_FORMAT);
```

Comment:
issue: I think `start` must be changed to `end`!

Reply:
Sooooo thank you😭